### ISSUE TITLE
code: 99

### ISSUE THREAD

**Issue by d3xt3r2909** [SourcePointUSA/ios-cmp-app#257]
**Describe the bug**
Getting code 99 all the time. There is no documentation of what this means, so I am not able to fix it.

**About your campaign (please complete the following information):**
 - Account Id: 1253
 - Property Id: 9036
 - Property Name: try! GDPRPropertyName("earliAudio"),
 - Campaign Env: .Public
 - PrivacyManager Id: 234511
 - Targeting Params (if any): NO
 - AuthId (if any): NO

** About your environment **
 - SDK version: iPhone xr - physically  device, iPhone - simulator
 - Device: [e.g. iPhone6] : iPhone xr - physically  device, iPhone - simulator
 - OS: [e.g. iOS8.1]: 14.1

** To Reproduce**
1. Add an example from readme with my information. 
2. Getting:
```Error: Optional(Something went wrong in the webview (code: 99, title: general error, stackTrace: Script error.))```

**Expected behavior**
Expecting to see "something" ;) 

**Additional context**
Basically, an issue is that when I integrate an example from README, I am getting this error message:

```Error: Optional(Something went wrong in the webview (code: 99, title: general error, stackTrace: Script error.))```

I believe that's something connected with my environment, but because of the leak of information, cannot figure out what's an issue.

**Comment by d3xt3r2909** [comment]
The issue at the end was in setup scenarios for my property. This was almost impossible to figure out without help from one of your ppl.  

Is it possible to have more details that can lead you to what's wrong in setup?

**Comment by atanurdemirci** [comment]
Hi @d3xt3r2909, check our UI docs here: https://documentation.sourcepoint.com/getting-started-in-the-ui-1/getting-started-in-the-ui 
I think the main issue has been the fact that you haven't set up the property yourself.
Let me know if you need anything further!

**Comment by d3xt3r2909** [comment]
Yes, agree. 

But usually, devs shouldn't set up those. But with great help from you, we figure out issue in the end. Anyhow, I suggest that documenting error codes at least would be helpful for public API.

Thanks once again
